src/Pages/Profile/admin/EditorForm.js

```diff
@@ -58,16 +73,16 @@ export default function EditForm (props) {
                   </option>
                 ))}
               </select>
-              <Button
-                type='button'
-                onClick={() => {
-                  props.setPagesPrinted(0)
-                }}
-                color='info'
-                style={{ marginTop: '5px' }}
-              >
-                Reset Pages!
-              </Button>
+              <FormGroup className='reset-pages-group' check inline>
+                <Label check id='reset-pages2'>
+                  Reset Pages!
+                </Label>
+                <Input
+                  type='checkbox'
+                  id='reset-pages'
+                  onClick={event => resetPagesClicked()}
```

Comment:
you likely can get rid of the functions above, as we already send a `setPagesPrinted` prop [here in AdminView](https://github.com/SCE-Development/Core-v4/blob/58537e2f6043b05b4a2e0b8a9831d8c6a0896790/src/Pages/Profile/admin/AdminView.jsx#L102)

i would add a hook inside this react component to track if the checkbox is clicked or not. then, instead of the funcitons above

```js
function submitClicked() {
  if (resetPages) {
    props.setPagesPrinted(0)
  }
  props.handleSubmissionToggle()
}
```

Reply:
where ```resetPages``` would be a boolean true/false